src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -24,7 +24,8 @@ export class LogStreamRegistry {
 
     public constructor(
         public readonly configuration: CloudWatchLogsSettings,
-        private readonly activeStreams: Map<string, CloudWatchLogsData> = new Map<string, CloudWatchLogsData>()
+        private readonly activeStreams: Map<string, CloudWatchLogsData> = new Map<string, CloudWatchLogsData>(),
+        private readonly activeTextEditors: Map<string, vscode.TextEditor> = new Map<string, vscode.TextEditor>()
```

Comment:
If two things go together, then put them together:
```ts
interface ActiveTab {
    stream: CloudWatchLogsData
    editor: vscode.TextEditor
}
```
Then use one map. This will obviously mean you'd have to change other code too.